cmd/cloud-api-adaptor/main.go

```diff
@@ -57,13 +57,13 @@ func (cfg *daemonConfig) Setup() (cmd.Starter, error) {
 		cmd.Parse("aws", os.Args[1:], func(flags *flag.FlagSet) {
 			flags.StringVar(&awscfg.AccessKeyId, "aws-access-key-id", "", "Access Key ID, defaults to `AWS_ACCESS_KEY_ID`")
 			flags.StringVar(&awscfg.SecretKey, "aws-secret-key", "", "Secret Key, defaults to `AWS_SECRET_ACCESS_KEY`")
-			flags.StringVar(&awscfg.Region, "aws-region", "ap-south-1", "Region")
+			flags.StringVar(&awscfg.Region, "aws-region", "", "Region")
 			flags.StringVar(&awscfg.LoginProfile, "aws-profile", "test", "AWS Login Profile")
 			flags.StringVar(&awscfg.LaunchTemplateName, "aws-lt-name", "kata", "AWS Launch Template Name")
 			flags.BoolVar(&awscfg.UseLaunchTemplate, "use-lt", false, "Use EC2 Launch Template for the Pod VMs")
 			flags.StringVar(&awscfg.ImageId, "imageid", "", "Pod VM ami id")
 			flags.StringVar(&awscfg.InstanceType, "instance-type", "t3.small", "Pod VM instance type")
-			flags.StringVar(&awscfg.SecurityGroupId, "securitygroupid", "", "Security Group Id to be used for the Pod VM")
+			flags.Var(&awscfg.SecurityGroupIds, "securitygroupids", "Security Group Ids to be used for the Pod VM, comma separated")
```

Comment:
Will this cause other providers code change also? comma separated array looks has less impact but seems the key changed from `securitygroupid` to `securitygroupids`.

Reply:
It won't have impact to other providers since the code is specific to aws. However the other providers can use the same logic to add support for inputs which can take multiple values.

Reply:
@huoqifeng hope this addresses your question ?